artifacthub/library/pod-security-policy/host-filesystem/1.1.0/samples/psp-host-filesystem/example_allowed.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-host-filesystem
+  labels:
+    app: nginx-host-filesystem-disallowed
```

Comment:
Do we need the app label? The allowed/disallowed is out of sync for a number of files.